blog/2021-02-21-webinar-GDPR-european-value-privacy-Yamato-Logistics-Japan.md

```diff
@@ -2,6 +2,7 @@
 slug: webinar-GDPR-european-value-privacy-Yamato-Logistics-Japan
 title: "'Background of GDPR and European Values on Privacy', a Consento webinar at ProjectDS"
 author: Yuko Fujisawa
+author_title: CMO
```

Comment:
What does CMO stand for?

Reply:
maybe asking @neo-wsbi what she thinks would be a good author title for her?

Reply:
I use CMO (Chief Marketing Officer) upon introducing myself to people :)

Reply:
CMO of Georepublic?

Reply:
The other titles are all functional, not hierarchical. Adding "Chief" would mean to reference the organization.

Reply:
I don't mind using a different title as long as it shows I'm in Marketing/Sales, but I need to use a title which shows the level of authority in the company for the sake of reaching out to people (especially on LinkedIn). Any ideas??

Reply:
My point being is that Consento is not a company and as such, it would be weird to have a title like CMO without referring to what. If you are the CMO of Georepublic that may be okay to state.